lib/Models/Terria.js

```diff
@@ -363,6 +364,16 @@ Terria.prototype.start = function(options) {
         corsProxy.proxyDomains.push.apply(corsProxy.proxyDomains, config.proxyDomains);
         corsProxy.baseProxyUrl = that.configParameters.corsProxyBaseUrl;
 
+        var promises = [];
+
+        promises.push(loadJson(cp.proxyableDomainsUrl).then(function(proxyableDomains) {
+            if (proxyableDomains.proxyAllDomains) {
+                corsProxy.isOpenProxy = true;
```

Comment:
Two minor thoughts here -
- Why use two different names here? I'd vote for using `corsProxy.proxyAllDomains`. The "open" terminology confused me at first - I thought it was referring to opening a file for reading.
- Why are we giving users the power to set this to true at all? Isn't that a bad idea?